types/ember__ordered-set/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "npm-naming": false // The ember addon package exports *build-time*, not *run-time* code.
```

Comment:
I don't see this rule disabled in the other ember packages. What is different about this one?

Reply:
In this case the Ember package is actually being published separately to npm, whereas most of the `@ember/*` packages do *not* have distinct npm packages and are resolved via Ber’s build process (and are actually part of `ember-source`), but in this case the package also has no export matching what dtslint expects because of how Ember addons work.